dascore/core/coords.py

```
@@ -841,7 +842,7 @@ def _slice_degenerate(self, sliz):
 
     def get_slice_tuple(
         self,
-        select: slice | None | type(Ellipsis) | tuple[Any, Any],
+        select: slice | None | EllipsisType | tuple[Any, Any],
```

Ruff (0.16.0)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)